product_docs/docs/pgd/5/durability/group-commit.mdx

```diff
@@ -108,5 +123,49 @@ transaction abort is requested. If the transaction is already
 decided to be committed at the time the abort request is sent, the transaction
 does eventually COMMIT even though the client might receive an abort message.
 
+See also [Limitations](limitations). 
+
+### Transaction Reconciliation
+
+A group commit transaction’s commit on the origin node is implicitly converted 
+into a two step commit. 
```

Comment:
@djw-m should we call this two phase commit and link to that definition? https://www.enterprisedb.com/docs/pgd/latest/terminology/#two-phase-commit-2pc

Reply:
Yes, to changing to two phase commit, no to linking back to the terminology as we're deep into the discussion of transactions and their resolutions at different visibilities and a link would be superfluous (and possibly irritating to a reader who sees the link, follows it and is taken to a super basic explanation of what a 2pc is)

Reply:
Have done changes to following text to make the 2pc nature more clear rather tha an abstract step.